assignments/arrays.js

```diff
@@ -63,32 +63,53 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
-
+for(i = 0; i < inventory.length; i++){
+    if(inventory[i].id === 33){
+        console.log('Car 33 is a', inventory[i]['car_year'], inventory[i]['car_make'], inventory[i]['car_model'] + '.');
+    }
+}
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+let lastCar = inventory[inventory.length-1];
+console.log(lastCar.car_make + " " + lastCar.car_model);
 
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
 let carModels = [];
-console.log();
+for (var i=0; i < inventory.length;i++){
```

Comment:
IMHO, using `var` in for loops is a risky proposition. Unlike `let` and `const`, `var` is only function-scoped, and will leak out of blocks like if statements and for loops. In this case, everything is fine because you continually reset `i` to zero, but be aware that `i` exists outside of the for loop because you used `var`, and depending on the circumstance or surrounding code, that may lead to unexpected behavior.

TLDR: I strongly suggest you use `const` and `let`.